### ISSUE TITLE
Parameter issues on training coco-data

### ISSUE THREAD

**Issue by athmey** [jugg1024/Text-Detection-with-FRCN#4]
Really appreciate your sharing about the FRCN.

Currently I am following your steps to train the coco-text data.

I use all of the default settings in your script to train the pretrained model, however the result is not so good as yours.(iteration times = 70000, other parameters are the same as that in faster_rcnn_end2end.yml)

Would you please share the essential parameters that you used to train the coco-fine-tuned-model?
such as iteration times,learning rate and some other parameters which should be modified. Thank you so much.

**Comment by athmey** [comment]
This is the final result that I got by using the default coco-text database

Evaluating detections
Writing text VOC results file
VOC07 metric? Yes
AP for text = 0.3453
Mean AP = 0.3453
~~~~~~~~
Results:
0.345
0.345

**Comment by jugg1024** [comment]
hi @athmey , i also use the default settings of frcn, but the pretrain model i use is official VGG16. And do you use full coco-text training data?
